samples/previous-viewpoint/src/services/customService/CustomService.ts

```diff
@@ -0,0 +1,47 @@
+import { ServiceBase } from "@vertigis/web/services";
+import { command, MapsLike, ViewpointLike } from "@vertigis/web/messaging";
+
+export default class CustomService extends ServiceBase {
+    private _history: ViewpointLike[] = [];
+    private movedBecauseCommand = false;
+
+    @command("custom-service.push-viewpoint")
+    protected _handlePushViewpoint(args: {
+        maps: MapsLike;
+        viewpoint: ViewpointLike;
+    }): void {
+        // Don't update the history if the reason the viewpoint changed was our
+        // `goToPreviousViewPoint` command
+        if (!this.movedBecauseCommand) {
```

Comment:
nit can we change the name of this maybe something like skipSaveNextViewpoint

Reply:
Updated.